### PR TITLE
UI fix for delete button highlight - 2456

### LINKED ISSUE #2456: 1.10 rc2 Issue - UI: Ticket Task Action Menu
image
image

### PR DESCRIPTION
**Pull request by nfebuary**
Fixes #2456

### CODE REVIEW

File: include/staff/templates/task-view.tmpl.php [hunk 1]
```diff
@@ -130,10 +130,9 @@ class="icon-reorder"></i> <?php
                 class="action-dropdown anchor-right">
                 <ul>
             <?php foreach ($actions as $a => $action) { ?>
-                    <li>
-                        <a class="no-pjax <?php
```
Comment (jdelhome3578) [inline include/staff/templates/task-view.tmpl.php]: Is `no-pjax` required here?

File: include/staff/templates/task-view.tmpl.php [hunk 2]
```diff
@@ -130,10 +130,9 @@ class="icon-reorder"></i> <?php
...
-                            echo $action['class'] ?: 'task-action'; ?>"
-                            <?php
+                    <li class="<?php
+                            echo $action['class'] ?: 'task-action'; ?>">
+                        <a class="no-pjax task-action" <?php
```
Comment (jdelhome3578) [inline include/staff/templates/task-view.tmpl.php]: Is `no-pjax` required here?

File: include/staff/templates/task-view.tmpl.php [hunk 3]
```diff
@@ -130,10 +130,9 @@ class="icon-reorder"></i> <?php
                 class="action-dropdown anchor-right">
                 <ul>
             <?php foreach ($actions as $a => $action) { ?>
-                    <li>
-                        <a class="no-pjax <?php
-                            echo $action['class'] ?: 'task-action'; ?>"
-                            <?php
+                    <li class="<?php
+                            echo $action['class'] ?: 'task-action'; ?>">
```
Comment (jdelhome3578) [inline include/staff/templates/task-view.tmpl.php]: Is `task-action` required?

File: include/staff/templates/task-view.tmpl.php [hunk 4]
```diff
@@ -130,10 +130,9 @@ class="icon-reorder"></i> <?php
                 class="action-dropdown anchor-right">
                 <ul>
             <?php foreach ($actions as $a => $action) { ?>
-                    <li>
-                        <a class="no-pjax <?php
-                            echo $action['class'] ?: 'task-action'; ?>"
-                            <?php
+                    <li class="<?php
```
Comment (jdelhome3578) [inline include/staff/templates/task-view.tmpl.php]: Would this be cleaner:
`<?php
                if ($action['class'])
                    echo sprintf("class='%s'", $action['class']); ?> >`

Comment (nfebuary) [inline include/staff/templates/task-view.tmpl.php]: Yes, that would be cleaner. I'll commit that change. Thanks!

**Pull request merged**